flow-defs/globals.js

```diff
@@ -0,0 +1,6 @@
+/* eslint-disable */
+
+declare var describe: Function
+declare var test: Function
+declare var it: Function
+declare var expect: Function
```

Comment:
mmh, do you have the **jest** type def in the `flow-typed` directory? it's supposed to define them, if not it feels it's a regression maybe 🤔

Reply:
you're right. works well after using `yarn flow-typed` (maybe we should put it on postinstall :trollface: )